build.rs

```diff
@@ -37,14 +37,38 @@
 
 extern crate gcc;
 use std::env;
+use std::process::Command;
+use std::collections::HashMap;
+
+fn run_lsb_release(args: Vec<&str>) -> HashMap<String, String> {
+    let output = Command::new("lsb_release")
+                         .args(args)
+                         .output()
+                         .expect("failed to run lsb_release");
+    assert!(output.status.success(), "lsb_release failed");
```

Comment:
Having one thing as an `expect` and one as an `assert` feels a bit odd? Maybe make both `expect`s and/-r unify the message? Or, to be honest, you could just `unwrap` them (I don't think putting `expect` everywhere is a good idea: it clogs up the source code).

Reply:
If you don't think it's favourable to include why we crashed, then we can just unwrap. Yeah. I got into this habit as backtraces rarely work(ed?) on BSD.

Reply:
I think they might start working once the Rust port properly moves to LLVM and libunwind is fixed. Anyway, up to you.